config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Oaisys::Engine.routes.draw do
+  resources :pmh, only: [:index]
```

Comment:
This isn't really a "RESTful" resource (the PMH protocol is really anti-REST in structure), so rather than start with RESTful resource routing and then restrict it to a single action, we should probably just do it directly via `match`: https://guides.rubyonrails.org/routing.html#non-resourceful-routes

And if we're going that route, we could use a more semantically meaningful name than "index" for this action, since it's not actually going to be an index in the normal REST/Rails sense.